api4/user_test.go

```diff
@@ -2451,6 +2451,39 @@ func TestRevokeAllSessions(t *testing.T) {
 	CheckUnauthorizedStatus(t, resp)
 }
 
+func TestRevokeSessionsFromAllUsers(t *testing.T) {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+
+	user := th.BasicUser
+	th.Client.Login(user.Email, user.Password)
+	_, resp := th.Client.RevokeSessionsFromAllUsers()
+	CheckForbiddenStatus(t, resp)
+
+	th.Client.Logout()
+	_, resp = th.Client.RevokeSessionsFromAllUsers()
+	CheckUnauthorizedStatus(t, resp)
+
+	admin := th.SystemAdminUser
+	th.Client.Login(admin.Email, admin.Password)
+	_, resp = th.Client.RevokeSessionsFromAllUsers()
+	CheckNoError(t, resp)
+
```

Comment:
```suggestion

// All sessions were revoked, so making the same call again will fail due to lack of a session.
```

Reply:
Suggesting this comment as it took me a couple of minutes to work out why this was expected to behave that way.